https-upgrades/README.md

```diff
@@ -53,4 +54,21 @@ for $testSet in test.json
     )
 
     expect($upgradedRequestURL === $test.expectURL)
-```
\ No newline at end of file
+```
+
+## Bloom filter sources
```

Comment:
bloom filter files are still not readable, but now we can include a source for them here